src/metacity-1/metacity-theme-3.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0"?>
 <metacity_theme>
 <info>
-	<name>Numix</name>
-	<author>Satyajit Sahoo</author>
```

Comment:
i think your changes are not big enough to discard Satya's authorship on the original theme